application setup instructions in the readme (#34)
econominhas/backend: README.md

		</tr>
	</tbody>
</table>

> ### SetUp

Reviewer: ```suggestion
### Running the API for the first time
```

> 1. Run <code>yarn start:dev</code>

Reviewer: Please add a step here to create a `.env` file based on the `.env.example`

> 1. Create <code>.env</code> copy and paste <code>.env.example</code>

Reviewer: ```suggestion
1. Copy and paste <code>.env.example</code> and rename the copy to <code>.env</code>
```

> 2. Run <code>yarn start:dev</code>

Reviewer: This should be done after running the migrations

> 2. Run <code>yarn start:dev</code>

Reviewer: Add another step before this one to run `yarn` and install the dependencies:
```
2. Run `yarn` to install the dependencies
```

   - Step 2.1: Run docker ps for get id container <code>econominhas-api</code>.

>    - Step 2.2: Run docker exec -it <code>\<ID container\></code>.

Reviewer: Run `yarn start:db` instead

>    - Step 2.3: Run <code>yarn db:migrate</code> to create the migrations.

Reviewer: Please, add a step to change the `.env` from:
```
DATABASE_URL=postgresql://username:password@postgres:5432/database?schema=public
```
to
```
DATABASE_URL=postgresql://username:password@localhost:5432/database?schema=public
```

And another one at the end to revert this change.

> 3. Run collections postman.

Reviewer: ```suggestion
3. Generate postman collections
```

> 4. Generate postman collections

Reviewer: Please, remove this step

>    - Setp 3.1: inside the econominhas-api container run <code>yarn openapi:postman</code>

Reviewer: Doesn't need to be inside the container, only needs to be in the terminal.

   - Step 3.2: import collection from <code>openapi/postman.json</code>

> 5. Available http://localhost:3000/v1

Reviewer: ```suggestion
5. Available at http://localhost:3000/v1
```

> 4. Available http://localhost:3000/

Reviewer: ```suggestion
4. Available http://localhost:3000/v1
```
